tests/unit/DocBlock/Tags/ReturnTest.php

```diff
@@ -152,7 +188,7 @@ public function testFactoryMethodFailsIfBodyIsNotString()
 
     /**
      * @covers ::create
-     * @expectedException \InvalidArgumentException
+     * @expectedException InvalidArgumentException
```

Comment:
PHPUnit ignores imported classes when resolving this, so leaving in the leading slash is less confusing for people, otherwise they may read it as being resolved as if it were imported.

Reply:
Restored slash